deployments/production/spark-master-controller.json

```diff
@@ -0,0 +1,84 @@
+{
+    "apiVersion": "extensions/v1beta1",
+    "kind": "Deployment",
+    "metadata": {
+        "labels": {
+            "component": "spark-cluster-master"
+        },
+        "name": "spark-master-controller",
+        "namespace": "spark-cluster-production"
+    },
+    "spec": {
+        "replicas": 1,
+        "selector": {
+            "matchLabels": {
+                "component": "spark-cluster-master"
+            }
+        },
+        "strategy": {
+            "rollingUpdate": {
+                "maxSurge": 1,
+                "maxUnavailable": 1
+            },
+            "type": "RollingUpdate"
+        },
+        "template": {
+            "metadata": {
+                "labels": {
+                    "component": "spark-cluster-master"
+                }
+            },
+            "spec": {
+                "containers": [
+                    {
+                        "args": [
+                            "master"
+                        ],
+                        "env": [
+                            {
+                                "name": "SPARK_MASTER_HOST",
+                                "value": "spark-cluster-master"
+                            },
+                            {
+                                "name": "SPARK_MASTER_PORT",
+                                "value": "7077"
+                            },
+                            {
+                                "name": "SPARK_MASTER_MEM",
+                                "value": "4G"
+                            }
+                        ],
+                        "image": "quay.io/winkapp/spark-standalone:1.6.0",
```

Comment:
@vellichor I don't but yeah let's move it to `master` now that the work is merged.